pkg/bazel/flags.pb.go

```diff
@@ -1,3 +1,11 @@
+/*
```

Comment:
Actually, this is generated code that is not covered by our license.

Reply:
yeah I know, but we don't assert that this matches the code generator so I think it's harmless and simpler to let the pre-commit check govern all .go files rather than try to exclude some.